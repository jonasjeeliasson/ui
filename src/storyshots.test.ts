// eslint-disable-next-line import/no-extraneous-dependencies
import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
// eslint-disable-next-line import/no-extraneous-dependencies
import styleSheetSerializer from 'jest-styled-components/src/styleSheetSerializer';
// eslint-disable-next-line import/no-extraneous-dependencies
import { addSerializer } from 'jest-specific-snapshot';

addSerializer(styleSheetSerializer);

initStoryshots({
  test: multiSnapshotWithOptions({}),
  storyNameRegex: /^(?!(Documentation|Experimental)).*$/,
});
