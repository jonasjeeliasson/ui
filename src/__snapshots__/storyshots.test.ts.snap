// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms | Box Different padding for different screen sizes  1`] = `null`;

exports[`Storyshots Atoms | Box Margin 1`] = `null`;

exports[`Storyshots Atoms | Box Margin and different margin Y-axis 1`] = `null`;

exports[`Storyshots Atoms | Box Padding  1`] = `null`;

exports[`Storyshots Atoms | Box Padding and different X-axis and left  1`] = `null`;

exports[`Storyshots Atoms | Card Basic Card 1`] = `null`;

exports[`Storyshots Atoms | Card Card rendered with different html tags 1`] = `null`;

exports[`Storyshots Atoms | CssGrid CssGrid with custom gutter 1`] = `null`;

exports[`Storyshots Atoms | CssGrid CssGrid with custom templateColumns 1`] = `null`;

exports[`Storyshots Atoms | CssGrid CssGrid with different layouts for different screen sizes 1`] = `null`;

exports[`Storyshots Atoms | CssGrid CssGrid with object as gutter and custom sized columns 1`] = `null`;

exports[`Storyshots Atoms | CssGrid Simple CssGrid 1`] = `null`;

exports[`Storyshots Atoms | Flag Available flags 1`] = `null`;

exports[`Storyshots Atoms | Flag Default use 1`] = `null`;

exports[`Storyshots Atoms | Flag Different size  1`] = `null`;

exports[`Storyshots Atoms | Flag Renders nothing if wrong country code 1`] = `null`;

exports[`Storyshots Atoms | Flexbox Flexbox default 1`] = `null`;

exports[`Storyshots Atoms | Flexbox Flexbox with conditionally hidden flex item 1`] = `null`;

exports[`Storyshots Atoms | Flexbox Flexbox with conditionally visible flex item 1`] = `null`;

exports[`Storyshots Atoms | Flexbox Flexbox with custom gutter 1`] = `null`;

exports[`Storyshots Atoms | Flexbox Flexbox with direction: column 1`] = `null`;

exports[`Storyshots Atoms | Icon Available icons 1`] = `null`;

exports[`Storyshots Atoms | Icon Default use 1`] = `null`;

exports[`Storyshots Atoms | Icon Different size and color 1`] = `null`;

exports[`Storyshots Atoms | List Basic ordered list 1`] = `null`;

exports[`Storyshots Atoms | List Basic unordered list 1`] = `null`;

exports[`Storyshots Atoms | ListItem Basic list item 1`] = `null`;

exports[`Storyshots Atoms | Media Basic usage 1`] = `null`;

exports[`Storyshots Atoms | Number Default 1`] = `null`;

exports[`Storyshots Atoms | Number Integration: with different typographies 1`] = `null`;

exports[`Storyshots Atoms | Number Invalid value 1`] = `null`;

exports[`Storyshots Atoms | Number Number ticks 1`] = `null`;

exports[`Storyshots Atoms | Number Number with 2 decimals  1`] = `null`;

exports[`Storyshots Atoms | Number Number with currency 1`] = `null`;

exports[`Storyshots Atoms | Number Number with percentage 1`] = `null`;

exports[`Storyshots Atoms | Number Number with sign 1`] = `null`;

exports[`Storyshots Atoms | Number null currency 1`] = `null`;

exports[`Storyshots Atoms | Separator Separator default 1`] = `null`;

exports[`Storyshots Atoms | Separator Separator vertical 1`] = `null`;

exports[`Storyshots Atoms | Spinner Colors 1`] = `null`;

exports[`Storyshots Atoms | Spinner Regression: 2 spinners with same color and size affects each other in Chrome 1`] = `null`;

exports[`Storyshots Atoms | Spinner Spinner big 1`] = `null`;

exports[`Storyshots Atoms | Spinner Spinner default 1`] = `null`;

exports[`Storyshots Atoms | TabTitle Active 1`] = `null`;

exports[`Storyshots Atoms | TabTitle Default 1`] = `null`;

exports[`Storyshots Atoms | TabTitle Integration: with Flexbox 1`] = `null`;

exports[`Storyshots Atoms | Table Default 1`] = `null`;

exports[`Storyshots Atoms | Table Integration: Table (stickyHeader) with Typography 1`] = `null`;

exports[`Storyshots Atoms | Table Integration: Table with Typography 1`] = `null`;

exports[`Storyshots Atoms | Table Integration: Table with Typography and ellipsis on name 1`] = `null`;

exports[`Storyshots Atoms | Time Invalid value 1`] = `null`;

exports[`Storyshots Atoms | Time Time  1`] = `null`;

exports[`Storyshots Atoms | Time Uses custom symbol for invalid value 1`] = `null`;

exports[`Storyshots Atoms | Typography Caption 1`] = `null`;

exports[`Storyshots Atoms | Typography Colors 1`] = `null`;

exports[`Storyshots Atoms | Typography Hero 1`] = `null`;

exports[`Storyshots Atoms | Typography Primary 1`] = `null`;

exports[`Storyshots Atoms | Typography Secondary 1`] = `null`;

exports[`Storyshots Atoms | Typography Tertiary 1`] = `null`;

exports[`Storyshots Atoms | Typography Title1 1`] = `null`;

exports[`Storyshots Atoms | Typography Title2 1`] = `null`;

exports[`Storyshots Atoms | Typography Title3 1`] = `null`;

exports[`Storyshots Molecules | Avatar Basic usage 1`] = `null`;

exports[`Storyshots Molecules | Avatar With element as children 1`] = `null`;

exports[`Storyshots Molecules | Button Button primary with size modified 1`] = `null`;

exports[`Storyshots Molecules | Button Button secondary with size modified 1`] = `null`;

exports[`Storyshots Molecules | Button Button that is full width 1`] = `null`;

exports[`Storyshots Molecules | Button Button with type modified 1`] = `null`;

exports[`Storyshots Molecules | Button Button with variant 1`] = `null`;

exports[`Storyshots Molecules | Button Buttons composed in a group 1`] = `null`;

exports[`Storyshots Molecules | Button Default usage 1`] = `null`;

exports[`Storyshots Molecules | Button Disabled button 1`] = `null`;

exports[`Storyshots Molecules | Button Link looking like a button 1`] = `null`;

exports[`Storyshots Molecules | Button Link looking like a button that is full width 1`] = `null`;

exports[`Storyshots Molecules | Button Link looking like a buttons composed in a group 1`] = `null`;

exports[`Storyshots Molecules | CardWithTitle CardWithTitle as article (most CardWithTitle should be article) 1`] = `null`;

exports[`Storyshots Molecules | CardWithTitle CardWithTitle as section 1`] = `null`;

exports[`Storyshots Molecules | CardWithTitle Default 1`] = `null`;

exports[`Storyshots Molecules | CardWithTitle Integration: CardWithTitle with custom component as title 1`] = `null`;

exports[`Storyshots Molecules | Development As currency 1`] = `null`;

exports[`Storyshots Molecules | Development As percentage 1`] = `null`;

exports[`Storyshots Molecules | Development Default 1`] = `null`;

exports[`Storyshots Molecules | Development Integration: with different typographies 1`] = `null`;

exports[`Storyshots Molecules | Development With icon 1`] = `null`;

exports[`Storyshots Molecules | LabeledValue Default 1`] = `null`;

exports[`Storyshots Molecules | LabeledValue Integration: LabeledValue secondary currency 1`] = `null`;

exports[`Storyshots Molecules | LabeledValue Integration: LabeledValue title1 1`] = `null`;

exports[`Storyshots Molecules | LabeledValue Integration: LabeledValue with negative development and currency 1`] = `null`;

exports[`Storyshots Molecules | LabeledValue Integration: LabeledValue with positive development and currency 1`] = `null`;

exports[`Storyshots Molecules | Link Default usage 1`] = `null`;

exports[`Storyshots Molecules | Link Link with _blank target 1`] = `null`;

exports[`Storyshots Molecules | LinkBuy LinkBuy 1`] = `null`;

exports[`Storyshots Molecules | LinkBuy LinkBuy disabled 1`] = `null`;

exports[`Storyshots Molecules | LinkSell LinkSell 1`] = `null`;

exports[`Storyshots Molecules | LinkSell LinkSell disabled 1`] = `null`;

exports[`Storyshots Molecules | ListWithTitles Integration: ListWithTitles with left and right titles 1`] = `null`;

exports[`Storyshots Molecules | ListWithTitles Integration: ListWithTitles with left title 1`] = `null`;

exports[`Storyshots Molecules | ListWithTitles Integration: ListWithTitles with no titles 1`] = `null`;

exports[`Storyshots Molecules | ListWithTitles Integration: ListWithTitles with right title 1`] = `null`;

exports[`Storyshots Molecules | PageWrapper PageWrapper 1`] = `null`;

exports[`Storyshots Molecules | PageWrapper PageWrapper with a custom background 1`] = `null`;

exports[`Storyshots Molecules | Tabs Default (extra space inside) 1`] = `null`;

exports[`Storyshots Molecules | TabsNav Integration: With react-router (extra space inside) 1`] = `null`;

exports[`Storyshots Organisms | CardWithTabs Integration: with Typography and extra space inside 1`] = `null`;

exports[`Storyshots Theme Colors (palette) 1`] = `null`;

exports[`Storyshots Theme Colors (semantic) 1`] = `null`;

exports[`Storyshots Theme Screen sizes 1`] = `null`;
